### DISCUSSION TITLE
What's the purpose of this repository?

### DISCUSSION THREAD

**Discussion by RS-gty** [flowerfalling/Deep-learning#2] [Q&A] [answered]
wondering(

**Comment by flowerfalling** [comment, marked as answer]
There are some examples of CNN and RNN in this repo. There are also some GAN demos (unfinished, so most of them cannot be used).
You can learn simple CNN and RNN through it or learn GAN together (mode collapse in GAN is a big problem).
In the Docker folder, you can deploy jupyter to run this project when you don't understand your configuration environment.
